.env.example

```
@@ -0,0 +1,66 @@
+# Number of Backup Files to Retain
+BACKUP_CRON_MAX_KEEP=7
+
+# Backup Storage to S3
+# Set to true to enable S3 storage
+BACKUP_ENABLE_S3=false
+BACKUP_S3_REGION="auto"
+BACKUP_S3_BUCKET="pdnode-backups"
+BACKUP_S3_ENDPOINT="https://your-id.r2.cloudflarestorage.com"
+BACKUP_S3_ACCESS_KEY="your-backup-key"
+BACKUP_S3_SECRET_KEY="your-backup-secret"
+BACKUP_S3_FORCE_PATH_STYLE=false
```

dotenv-linter (4.0.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)